components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/impl/rdbmssingleentry/RdbmsSingleEntrySessionDataStore.java

```diff
@@ -225,6 +225,12 @@ private String replaceTableName(String query) {
         return query;
     }
 
+    @Override
+    public String getStoreName() {
+
+        return "RDBMS_SingleEntry";
```

Comment:
Lets use the camel case for naming. 

`RDBMSSingleEntrySessionDataStore`. do not directly use this, instead create a constant for this and use that. We need to use that constant as a reference in multiple places

Reply:
Another thought. Since this is the better session data store that we want to use, let's change the name to `RDBMSSessionDataStore`.